contracts/paymaster/src/forward_call.rs

```diff
@@ -33,18 +33,25 @@ pub trait ForwardCall {
         let initial_payments = self.call_value().all_esdt_transfers();
```

Comment:
rename to callback_payments. Also add a TODO to use ManagedGetBackTransfers here once rc1.6 is activated.

Reply:
Done